src/Sylius/Bundle/AdminBundle/Grid/ProductAttributeGrid.php

```diff
@@ -0,0 +1,138 @@
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+
+final class ProductAttributeGrid extends AbstractGrid
+{
+    public function __construct(
+        private readonly string $productAttributeClass,
+    ) {
+    }
+
+    public static function getName(): string
+    {
+        return 'sylius_admin_product_attribute';
+    }
+
+    public function buildGrid(GridBuilderInterface $gridBuilder): void
+    {
+        $gridBuilder
+            ->setDriverOption('class', $this->productAttributeClass)
+            ->setRepositoryMethod('createListQueryBuilder', [
+                "expr:service('sylius.context.locale').getLocaleCode()",
+            ])
+            ->setLimits([10, 25, 50])
+            ->addOrderBy('position', 'asc')
+
+            // -- Fields
+            ->addField(
+                TwigField::create('name', '@SyliusAdmin/shared/grid/field/name.html.twig')
+                    ->setLabel('sylius.ui.name')
+                    ->withOptions([
+                        'vars' => [
+                            'th_class' => 'w-75',
+                        ],
+                    ]),
+            )
+            ->addField(
+                TwigField::create('code', '@SyliusAdmin/shared/grid/field/code.html.twig')
+                    ->setLabel('sylius.ui.code')
+                    ->setSortable(true)
+                    ->withOptions([
+                        'vars' => [
+                            'th_class' => 'w-25',
+                        ],
+                    ]),
+            )
+            ->addField(
+                TwigField::create('type', '@SyliusUi/grid/field/label.html.twig')
+                    ->setLabel('sylius.ui.type')
+                    ->setSortable(true),
+            )
+            ->addField(
+                TwigField::create('translatable', '@SyliusAdmin/shared/grid/field/boolean.html.twig')
+                    ->setLabel('sylius.ui.translatable')
+                    ->setSortable(true),
+            )
+            ->addField(
+                TwigField::create('position', '@SyliusUi/grid/field/position.html.twig')
+                    ->setLabel('sylius.ui.position')
+                    ->setSortable(true)
+                    ->withOptions([
+                        'vars' => [
+                            'th_class' => 'w-1 text-center',
+                            'td_class' => 'text-center',
+                        ],
+                    ]),
+            )
+
+            // -- Filter
+            ->addFilter(
+                StringFilter::create('search', ['translations.name', 'code'])
+                    ->setLabel('sylius.ui.search')
+                    ->setFormOptions(['type' => StringFilter::TYPE_CONTAINS]),
+            )
+            ->addFilter(
+                SelectFilter::create('type', [
+                    'sylius.ui.checkbox' => \Sylius\Component\Attribute\AttributeType\CheckboxAttributeType::TYPE,
```

Comment:
Could you simplify this?
```suggestion
                    'sylius.ui.checkbox' => CheckboxAttributeType::TYPE,
```